product_pack_category_restriction/tests/test_product_pack_product_category.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.exceptions import ValidationError
+from odoo.tests import TransactionCase
+
+
+class TestProductPackCategoryRestriction(TransactionCase):
+    def setUp(self):
+        super().setUp()
```

Comment:
Change to setUpClass according to Migration Rules v15

![image](https://github.com/OCA/product-pack/assets/123451545/085f8d8f-9c42-42cd-b868-0e0aa28266eb)

Reply:
its updated,
Thanks